dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -121,7 +121,7 @@ public void createTest() throws Exception {
             getClient(authToken).perform(post("/api/core/communities")
                                         .content(mapper.writeValueAsBytes(comm))
                                         .contentType(contentType)
-                                .param("projection", "full"))
+                                   .param("embed", CommunityMatcher.getFullEmbedsParameters()))
```

Comment:
There isn't really a "problem", but we aren't testing for a full projection. I would keep the full projection for a single test where you actually test it as the current test is passing with the "level" projection.

Reply:
if not necessary please revert it otherwise add at least a test for the full projection or copy the original test in an ignored test and report about it in a jira ticket